app/assets/stylesheets/defaults.scss

```diff
@@ -0,0 +1,11 @@
+nav {
+  ul {
+    a {
+      font-size: 1.6rem;
+    }
+  }
+}

```

Comment:
I can, but the whole point was that it should be specifically for that. It only should affect elements inside the `nav`. `nav{ul{a}}` is functionally the same as `nav{a}` because there will never be a case where `<a>` is outside a `<ul>` in the `<nav>`  My plan is that it is brittle on purpose, because I only want that element sized so big.

Reply:
I just looked at the styles that get applied, and saw the selector you are fighting.  This materialize library is ass, and I expect we will increasingly have to hack around it like this.  We should do our best not to get too tangled up in it.  However, I think this is a necessary hack for the time being.